thirdparty/github.go

```diff
@@ -541,3 +542,73 @@ func CheckGithubAPILimit(ctx context.Context, oauthToken string) (int64, error)
 
 	return int64(limits.Core.Remaining), nil
 }
+
+// GithubUserInOrganization returns true if the given github user is in the
+// given organization. The user with the attached token must have
+// visibility into organization membership, including private members
+func GithubUserInOrganization(ctx context.Context, token, requiredOrganization, username string) (bool, error) {
```

Comment:
should we also clarify the three strings in this function too?

Reply:
1. smoke-test-task is being weird, Brian has a ticket
2. The fact that patch.GithubPatch existed already meant I was happy to use it. In this case, I'm less inclined to do so. I don't like making structs just to pass arguments in, and I'm leaning toward leaving this one alone.